test/dlc_tests/conftest.py

```diff
@@ -176,17 +185,19 @@ def ec2_instance(
                 LOGGER.error(f"Failed to launch in {a_zone} with Error: {e}")
                 continue
     else:
-        instances = ec2_resource.create_instances(**params)
+        try:
+            instances = ec2_resource.create_instances(**params)
+        except ClientError as e:
+            if "InsufficientInstanceCapacity" in str(e):
```

Comment:
Instead of string checking, I believe the appropriate way to look for the error code is:
```
if e.response['Error']['Code'] == "InsufficientInstanceCapacity":
```

Reply:
Sure, this makes sense. I'll update it. Thanks for the recommendation.